spec/jobs/cron/send_api_token_expiration_notice_job_spec.rb

```diff
@@ -0,0 +1,68 @@
+RSpec.describe Cron::SendAPITokenExpirationNoticeJob, type: :job do
+  describe 'perform' do
+    let(:administrateur) { create(:administrateur) }
+    let!(:token) { APIToken.generate(administrateur).first }
+    let(:mailer_double) { double('mailer', deliver_later: true) }
+    let(:today) { Time.zone.local(2018, 01, 01).to_date }
+
+    def perform_now
+      Cron::SendAPITokenExpirationNoticeJob.perform_now
+    end
+
+    before do
+      Timecop.freeze(today)
+      allow(APITokenMailer).to receive(:expiration).and_return(mailer_double)
+    end
+
+    after { Timecop.return }
```

Comment:
```suggestion
    before do
      travel_to today
      allow(APITokenMailer).to receive(:expiration).and_return(mailer_double)
    end
```
(pas d'after, rails appelle `travel_back` tout seul)